analyses/focal-cn-file-preparation/README.md

```diff
@@ -17,26 +15,31 @@ bash analyses/focal-cn-file-preparation/run-prepare-cn.sh
+* `01-add-ploidy-cnvkit.Rmd` - The two CNV callers, CNVkit and ControlFreeC, do not handle ploidy in the same way ([A Note on Ploidy](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/de661fbe740717472fcf01c7d9b74fe1b946aece/doc/data-formats.md#a-note-on-ploidy) in the Data Formats documentation). 
   This notebook adds the ploidy inferred via ControlFreeC to the CNVkit data and adds a status column that defines gain and loss broadly.
   (Note that [the logic around sex chromosomes in males when ploidy = 3 leaves something to be desired](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/259#discussion_r345354403)).
-* `01-prepare-cn-file.R` - This script performs the ranges to annotation mapping using the GENCODE v27 GTF included via the data download step; it takes the ControlFreeC file and the CNVkit file prepared with `00-add-ploidy-cnvkit.Rmd` as input.
-  The mapping is limited to _coding sequences_.
-  Mapping to cytobands is performed with the [`UCSC hg38 cytoband file`](http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz).
-  _Note: The decision to implement the `UCSC file` was made based on a comparison done between the cytoband calls in the `org.Hs.eg.db` package and the calls in the `UCSC file`. We found that they disagreed in ~11,800 calls out of ~800,000 and the `UCSC file` contains more cytoband calls._
+* `02-add-ploidy-consensus.Rmd` - This is very similar to the CNVkit file prep.
+However, there are instances in the consensus SEG file where `copy.num` is `NA` which are removed.
+See the notebook for more information.
+* `03-prepare-cn-file.R` - This script performs the ranges to annotation mapping using the GENCODE v27 GTF included via the data download step; it takes the ControlFreeC file or a SEG (e.g., CNVkit, consensus SEG) file prepared with `00-add-ploidy-cnvkit.Rmd` as input.
+  The mapping is limited to _exons_. 
+  Mapping to cytobands is performed with the [`org.Hs.eg.db`](https://doi.org/doi:10.18129/B9.bioc.org.Hs.eg.db) package.
   A table with the following columns is returned:
   
-  | biospecimen_id | arm | broad_status | status | copy_number | ploidy | ensembl | gene_symbol |
-  |----------------|-----|--------------|--------|-------------|--------|---------|-------------|
+  | biospecimen_id | status | copy_number | ploidy | ensembl | gene_symbol | cytoband |
+  |----------------|--------|-------------|--------|---------|-------------|---------|
   
-* `02-rna-expression-validation.R` - This script examines RNA-seq expression levels (RSEM FPKM) of genes that are called as deletions.
+* `rna-expression-validation.R` - This script examines RNA-seq expression levels (RSEM FPKM) of genes that are called as deletions. 
+It is not currently run via the shell script.
```

Comment:
What's the plan for this script then? Is it being dropped or revamped or is it supposed to be called somewhere else?

Reply:
Plan is to rework it #387